### PR TITLE
Pcvl 774 promote display config to a persistent config

### PR DESCRIPTION
**Pull request by MarionQuandela**

### CODE REVIEW

File: perceval/rendering/circuit/display_config.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    if PDISPLAY_KEY in config:
+        skin = config[PDISPLAY_KEY].get(SKIN_KEY, "PhysSkin")
+        if skin not in SKIN_MAPPING:
+            get_logger().error(f"Invalid skin in persistent configuration {skin}")
+            skin = "PhysSkin"
+    return SKIN_MAPPING[skin]
+
+
+class DisplayConfig:
+    _selected_skin = _get_default_skin()
```
Comment (raksharuia) [inline perceval/rendering/circuit/display_config.py:61]: Include documentation.

File: perceval/rendering/circuit/display_config.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    @staticmethod
+    def select_skin(skin: type[ASkin]) -> None:
+        DisplayConfig._selected_skin = skin
+
+    @staticmethod
+    def get_selected_skin(**kwargs) -> ASkin:
+        return DisplayConfig._selected_skin(**kwargs)
+
+    @staticmethod
+    def save_select_skin() -> type:
```
Comment (raksharuia) [inline perceval/rendering/circuit/display_config.py:72]: update name to save

**Review by raksharuia — CHANGES_REQUESTED** [review]

File: docs/source/reference/pdisplay.rst [hunk 3]
```diff
@@ -47,27 +52,46 @@ Then, add ``pcvl.pdisplay()`` of your circuit.
...
+  pcvl.pdisplay(C, skin=symbolic_skin)
 
 .. figure:: ../_static/img/decomposition_symb_compact_false.png
   :align: center
   :width: 25%
 
+By default the skin will be ``PhysSkin``, if you want to use another skin by default, you can save your configuration into your Perceval persistent configuration.
+
+In order to do that you need to use the object ``DisplayConfig`` like that:
+
```
Comment (raksharuia) [inline docs/source/reference/pdisplay.rst:82]: To save configuration for your choice of skin by default, you require the `DisplayConfig` object. Usage example -

File: perceval/rendering/circuit/display_config.py [hunk 4]
```diff
@@ -58,18 +58,32 @@ def _get_default_skin():
 
 
 class DisplayConfig:
+    """Handle the display configuration such as:
+
+        - Skin use for pdisplay. Default skin is the one in the persistent data or, if no config is find, PhysSkin.
```
Comment (raksharuia) [inline perceval/rendering/circuit/display_config.py:63]: .... if no config is fOUnd, ...

File: perceval/rendering/circuit/display_config.py [hunk 5]
```diff
@@ -58,18 +58,32 @@ def _get_default_skin():
...
+    """
     _selected_skin = _get_default_skin()
 
     @staticmethod
     def select_skin(skin: type[ASkin]) -> None:
+        """Select the skin pdisplay uses
```
Comment (raksharuia) [inline perceval/rendering/circuit/display_config.py:69]: used by pdisplay

File: docs/source/reference/pdisplay.rst [hunk 6]
```diff
@@ -47,27 +52,46 @@ Then, add ``pcvl.pdisplay()`` of your circuit.
...
+.. code-block::
+
+  from perceval.rendering.circuit import DisplayConfig, SymbSkin
```
Comment (ericbrts) [inline docs/source/reference/pdisplay.rst:85]: Should we expose `DisplayConfig` and skin classes directly from `perceval.rendering` to simplify this import? (I'm for it)

**Review by raksharuia — APPROVED** [review]

**Pull request merged**